screens/CreateEventScreen.jsx

```diff
@@ -0,0 +1,414 @@
+                <Switch
+                  style={styles.formResponseContainer}
+                  value={hearingAssistance}
+                  onValueChange={toggleHearingAssistance}
+                />
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.formSectionContainer}>
+            <Text style={styles.formSectionText}>Logistical Information</Text>
+            <View style={styles.formBlockContainer}>
+              <Text style={styles.formHeaderText}>Minimum Ticket Price</Text>
+              <View style={styles.formResponseContainer}>
+                <TextInput
+                style={styles.formResponseText}
+                placeholder="e.g. $25.00"
+                value={ticketPrice}
+                textAlign={'left'}
+                onChangeText={(text) => setTicketPrice(text)}
+                />
+              </View>
+            </View>
+            <View style={styles.formBlockContainer}>
+              <Text style={styles.formHeaderText}>Ticket Link</Text>
+              <View style={styles.formResponseContainer}>
+                <TextInput
+                style={styles.formResponseText}
+                placeholder="e.g. https://example.com"
+                value={ticketLink}
+                textAlign={'left'}
+                onChangeText={(text) => setTicketLink(text)}
+                />
+              </View>
+              <View style={styles.buttonRow}>
+                <View style={styles.formButtonContainer}>
+                  <Text style={styles.formHeaderText}>Intermission?</Text>
+                  <Switch
+                    style={styles.formResponseContainer}
+                    value={intermission}
+                    onValueChange={toggleIntermission}
+                  />
+                </View>
+                <View style={styles.formButtonContainer}>
+                  <Text style={styles.formHeaderText}>Ticket Required?</Text>
+                  <Switch
+                    style={styles.formResponseContainer}
+                    value={ticketRequired}
+                    onValueChange={toggleTicketRequired}
+                  />
+                </View>
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.formSectionContainer}>
+            <Text style={styles.formSectionText}>Cast & Creatives</Text>
+            <View style={styles.container}>
+            <View style={styles.formResponseContainer}>
+              <TextInput
+                placeholder="Name here"
+                style={styles.formResponseText}
+                textAlign={'left'}
+                value={cast[castId].name}
+                onChangeText={(text)=>fetchUsers(text) ? fetchUsers(text) : onChange(text, 'name')} />
+              <FlatList
+                style={styles.userList}
+                data={users}
+                renderItem={({ item, index }) => addUserProfile(item)}
+              />
```

Comment:
honestly, unsure what the best approach is here. we can have a flatlist like we are currently, or a tooltip that pops up saying (do you mean <user> ?) with their picture if there is an exact match. tbh, feel like this situation of tagging an existing user is fairly niche??? so i wouldn't want it to be too intrusive.

Reply:
would love to hear your thoughts on this. might need to talk to jasmin/eva

Reply:
might need a custom textinput object that can return a textinput & absolutely positioned flatlist combo. not sure we can achieve the desired effect like this. maybe create that and import it in the cast component?